pkg/storage/unified/resource/hooks.go

```diff
@@ -8,10 +8,6 @@ import (
 )
 
 type WriteAccessHooks struct {
-	// Check if a user has access to write folders
-	// When this is nil, no resources can have folders configured
-	Folder func(ctx context.Context, user claims.AuthInfo, uid string) bool
```

Comment:
🤔  I think we still need this.  We need to be able to identify if a resource should be allowed to live in a folder or not (eg dashboards and folders YES, playlists NO).